ggshield/cmd/sca/scan/ci.py

```diff
@@ -64,15 +68,39 @@ def scan_ci_cmd(
             return output_handler.process_scan_all_result(scan)
 
         check_git_dir()
-        commit_count = len(
-            collect_commit_range_from_ci_env(config.user_config.verbose)[0]
-        )
+
+        new_commits, _ = collect_commit_range_from_ci_env(config.user_config.verbose)
+        previous_commit, _ = get_previous_commit_from_ci_env(config.user_config.verbose)
+
+        # If no sha provided, consider repo was empty before
+        if previous_commit == EMPTY_SHA or previous_commit == "null":
```

Comment:
In what conditions can we receive a `"null"` string? Can we return `None` instead? It seems to me that turning `"null"` into `None` should be the job of the `*_from_ci_env()` functions.

Maybe the `*_from_ci_env()` functions can also turn `EMPTY_SHA` into `None`?